tests/startup/test_cli.py

```diff
@@ -151,3 +153,60 @@ def test_enterprise_disabled(self):
                 ''')
             # MQTT
             self.assert_mqtt_port(lambda x: x > 0)
+
+    def test_startup_logs(self):
+
+        # Create CRATE_HOME directory
+        tmp_home = self.mkdtemp()
+        Path(tmp_home, 'config').mkdir()
+
+        nodeName = self.fake.last_name()
+        settings = dict({
+            'node.name': nodeName,
+            'path.home': tmp_home,
+        })
+
+        logFilePath = Path(tmp_home, 'crate.log')
+        # Copy log4j2.properties and replace path to crate.log output file
```

Comment:
could also extract this in a method `create_log_from_template` to make the comment unnecessary